bin/clean_topics_vs_practices.py

```diff
@@ -0,0 +1,28 @@
+#!python3
```

Comment:
```suggestion
#!/usr/bin/env python3
```

We shouldn't assume that `$PATH` exists or is sane.

Reply:
Good point.